app/mailers/authentication_mailer.rb

```diff
@@ -5,9 +5,14 @@ class AuthenticationMailer < ::Devise::Mailer
 
   def confirmation_instructions(record, token, _opts = {})
     confirmation_link = confirmation_url(record, confirmation_token: token)
+    template_id = GOV_NOTIFY_CONFIG['confirmation_email']['template_id']
 
     SendConfirmationEmail.new(
       notifications_gateway: EmailGateway.new
-    ).execute(email: record.email, confirmation_url: confirmation_link)
+    ).execute(
+      email: record.email,
+      confirmation_url: confirmation_link,
+      template_id: template_id
+    )
```

Comment:
We could use `GOV_NOTIFY_CONFIG` directly in the Use Case but that would couple it to Rails. So we don't do it.